schematic/manifest/generator.py

```diff
@@ -1102,10 +1103,11 @@ def _create_requests_body(
         requests_body = {}
         requests_body["requests"] = []
         for i, req in enumerate(ordered_metadata_fields[0]):
-            # Gather validation rules and valid values for attribute
-            validation_rules = self.sg.get_node_validation_rules(req)
+            # Gather validation rules and valid values for attribute, if using google sheets.
 
-            if validation_rules:
+            validation_rules = self.sg.get_node_validation_rules(req)
+            
```

Comment:
I think it will also be worth adding some comments here to explain that we are only adding regex match to google sheet

Reply:
Done!